### PR TITLE
Fix startup failure

### PR DESCRIPTION
**Pull request by Half-Shot**
This fixes a crash on startup due to a change in how matrix-appservice-bridge works, that evaded our testing entirely.

### CODE REVIEW

File: src/Program.ts [hunk 1]
```diff
@@ -55,7 +56,8 @@ class Program {
                 try {
                     await this.runBridge(port, config as ConfigValue);
                 } catch (ex) {
-                    log.error("Failed to start:", ex);
+                    console.log(ex);
```
Comment (jaller94) [inline src/Program.ts]: Do you really want to use `console` here??

File: src/Metrics.ts [hunk 2]
```diff
@@ -14,10 +14,8 @@ interface IBridgeGauges {
 
 export class Metrics {
     public static init(bridge: Bridge) {
-        this.metrics = new PrometheusMetrics();
-        this.metrics.registerMatrixSdkMetrics();
+        this.metrics = bridge.getPrometheusMetrics()
```
Comment (jaller94) [inline src/Metrics.ts]: ChristianLint: :upside_down_face: 
```suggestion
        this.metrics = bridge.getPrometheusMetrics();
```

**Review by jaller94 — APPROVED** [review]
I'm only approving this because I don't expect this to break.
But it looks like a Work In Progress PR with the use of `console.log` and commented out `log.` line.

File: src/Metrics.ts (hunk 2 above)
Comment (Half-Shot) [reply src/Metrics.ts]: We need you as a service :D

**Pull request merged**